google/cloud/documentai_toolbox/converters/config/bbox_conversion.py

```diff
@@ -130,27 +133,19 @@ def _convert_bbox_units(
             A converted coordinate.
 
     """
-    final_coordinate = coordinate
-    if input_bbox_units != "normalized":
-        if input_bbox_units == "pxl":
-            if width is None:
-                final_coordinate = _normalize_coordinates(coordinate, height)
-            else:
-                final_coordinate = _normalize_coordinates(coordinate, width)
-        if input_bbox_units == "inch":
-            x = _convert_to_pixels(coordinate, 96)
-            if width is None:
-                final_coordinate = _normalize_coordinates(x, height)
-            else:
-                final_coordinate = _normalize_coordinates(x, width)
-        if input_bbox_units == "cm":
-            x = _convert_to_pixels(coordinate, 37.795)
-            if width is None:
-                final_coordinate = _normalize_coordinates(x, height)
-            else:
-                final_coordinate = _normalize_coordinates(x, width)
-
-    return final_coordinate * multiplier
+    pixel_conversion_rates = {
+        "pxl": 1,
+        "inch": 96,
+        "cm": 37.795,
+    }
```

Comment:
Can we put these in constants?

Reply:
Do you mean make the whole variable a constant? Or constants for the strings/Numbers?